_releases/2022-07-06-1.5.0-released.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Crystal 1.5.0 is released!
+version: 1.5.0
+summary:
+thumbnail: +
+author:
+---
+
+We are delivering a new release with several bugfixes and improvements. Below we list the most important or interesting changes, without mentioning several bugfixes and smaller enhancements. For more details, visit the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.5.0). Breaking changes are marked with ⚠️.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.5.0) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+## Stats
+
+In this release we included [XXX changes since the 1.4.1 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.5.0) by XX contributors. We thank all the effort put into improving the language! ❤️
+
+## Remarkable changes
+
+### Method restrictions from instance variables
+
+When an instance variable is assigned the value of an untyped method argument, then the argument is restricted to share the same type as the instance variable.
+
+For instance, consider this code:
+
+```crystal
+class Foo
+  @x : Int64
+
+  def initialize(x)
+    @x = x
+  end
+end
+````
+
+Up to 1.4.1, `x` in `initialize` is unrestricted. This had several issues:
+
+ 1. If the user passes an incorrect argument, like `Foo.new('a')`, then the error wrongly blames `@x` for not having the right type.
```

Comment:
I wouldn't say this is "wrong" - it's just part of the story. I think we could mention that the reason for error was hard to understand and the new feature makes it easier to work with instance var assignments.

Reply:
You're correct. I was misunderstanding the message, which is anyway plain wrong:

> Error: instance variable '@x' of Foo must be Int32, not Char